lib/ProjectContent.ts

```diff
@@ -0,0 +1,44 @@
+import { OctokitAction } from "./OctokitAction";
+import { components } from "@octokit/openapi-types/types.d";
+import { IssueOrPR } from "./IssueOrPR";
+
+export class ProjectContent {
+
+    private readonly action: OctokitAction;
+    private readonly columns: number[] = [];
+
+    private constructor(action: OctokitAction, columns: number[]) {
+        this.action = action;
+        this.columns = columns;
+    }
+
+    public static async FromColumn(action: OctokitAction, column_id: number): Promise<ProjectContent> {
+        const { data: column } = await action.rest.projects.getColumn({ column_id });
+        const project_id = parseInt(column.project_url.split("/").pop());
+        const { data: columns } = await action.rest.projects.listColumns({ project_id }); 
+        return new ProjectContent(action, columns.map(x => x.id));
+    }
+
+    public async findCard(issueOrPR: IssueOrPR): Promise<any> {
+        // We should start caching these results in case we need to call it multiple times from a single action
```

Comment:
Any reason not to do this now?

Reply:
Not to overengineer it, we don't need it (yet). And I'm not sure if we will any time soon.